README.md

```diff
@@ -1,12 +1,11 @@
-packager
-========
+# GOV.UK packaging
 
-This repository contains some scripts to aid the building of debian source
-packages (.dsc) files for submission to Launchpad or another build server for
-building into binary debian packages (.deb)
+This repo creates packages for GOV.UK.
 
-Usage
------
+## fpm
 
-See the ["Getting Started" documentation at
-`doc/getting_started.md`](doc/getting_started.md).
+This is the way we create packages now. It is easier than using debuild.
```

Comment:
I've added a little bit to the README but I don't want to let my opinions and bad experiences into the history of this repo too much :)